### ISSUE TITLE
Couldn't start it due to TypeError: NoneType takes no arguments

### ISSUE THREAD

**Issue by lwcorp** [firstoxe/TAPI-Event-monitor#1]
In Python 3.11.4, `python main.py` produces:
```
Traceback (most recent call last):
  File "main.py", line 13, in <module>
    class TapiEvents(win32.getevents(cls)):
TypeError: NoneType takes no arguments
```
Can anything be done?

**Comment by abdalkreemorabi** [comment]
same here, did you find any solution?

**Comment by lwcorp** [comment]
> same here, did you find any solution? @lwcorp

Yes, but only by doing my own project from scratch: https://github.com/lwcorp/lwtapi - in fact even as a simple single file.

**Comment by abdalkreemorabi** [comment]
Thanks a lot ! @lwcorp 
The issue I faced was with converting the script to `.exe` app using [Pyinstaller](https://pyinstaller.org/en/stable/)
After running the app I faced the error
I will try this https://github.com/lwcorp/lwtapi
